src/data/seeders/20180928110125-insert-registry.js

```diff
@@ -1,9 +1,12 @@
 'use strict'
 
+const constants = require('../constants')
+
 module.exports = {
   up: (queryInterface, Sequelize) => {
     return queryInterface.bulkInsert('Registries', [
       {
+        id: constants.DOCKER_REGISTRY_ID,
```

Comment:
Im not a huge fan of modifying preexisitng seeders and migrations. It could lead to consistency issues.

What if someone already has a DB, and just do an update ?